libsnowflakeclient/deps-build/darwin/aws/lib/cmake/aws-cpp-sdk-core/aws-cpp-sdk-core-targets.cmake

```diff
@@ -56,7 +56,7 @@ add_library(aws-cpp-sdk-core STATIC IMPORTED)
 set_target_properties(aws-cpp-sdk-core PROPERTIES
   INTERFACE_COMPILE_DEFINITIONS "AWS_SDK_VERSION_MAJOR=1;AWS_SDK_VERSION_MINOR=3;AWS_SDK_VERSION_PATCH=50"
   INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
-  INTERFACE_LINK_LIBRARIES "pthread;/Users/kristopherwagner/libsnowflakeclient/deps-build/darwin/curl/lib"
+  INTERFACE_LINK_LIBRARIES "pthread;/Users/bamboo/Desktop/harry/libsnowflakeclient/scripts/../deps-build/darwin/Release/curl/lib"
```

Comment:
I see the old files also has static paths? Are these files generated? Should they be in the repo at all?